content/guides/docker-concepts/running-containers/multi-container-applications.md

```diff
@@ -0,0 +1,203 @@
+    ```
+
+7. If you look at the Docker Dashboard, you can see the containers and dive deeper into their configuration.
+
+
+
+8. With everything up and running, you can open [http://localhost](http://localhost) in your browser to see the site. Refresh the page several times to see the host that’s handling the request and the total number of requests:
+
+    ```console
+    web2: Number of visits is: 9
+    web1: Number of visits is: 10
+    web2: Number of visits is: 11
+    web1: Number of visits is: 12
+    ```
+
+    > You might have noticed that Nginx, acting as a reverse proxy, likely distributes incoming requests in a round-robin fashion between the two backend containers. This means each request might be directed to a different container (web1 and web2) on a rotating basis. The output shows consecutive increments for both the web1 and web2 containers and the actual counter value stored in Redis is updated only after the response is sent back to the client.
+
+9. You can use the Docker Dashboard to remove the containers by selecting the containers and selecting the *Delete* button
+
+
+    ![A screenshot of Docker Dashboard showing multi-container applications](images/multi-container-apps.webp?border=true)
+
+## Simplify the deployment using Docker Compose
+
+
+Docker Compose provides a structured and streamlined approach for managing multi-container deployments. As stated earlier, with Docker Compose, you don’t need to run multiple `docker run` commands. All you need to do is define your entire multi-container application in a single YAML file called `compose.yml`. Let’s see how it works.
+
+
+Navigate to the root of the project directory. Inside this directory, you'll find a file named `compose.yml`. This YAML file is where all the magic happens! It defines all the services that make up your application, along with their configurations. Each service specifies its image, ports, volumes, networks, and any other settings necessary for its functionality.
+
+1. Use the `docker compose up` command to start the application:
+
+    ```console
+    $ docker compose up -d --build
+    ```
+
+    When you run this command, you should see output similar to the following:
+
+
+    ```console
+    Running 5/5
+    ✔ Network nginx-nodejs-redis_default    Created                                                0.0s
+    ✔ Container nginx-nodejs-redis-web1-1   Started                                                0.1s
+    ✔ Container nginx-nodejs-redis-redis-1  Started                                                0.1s
+    ✔ Container nginx-nodejs-redis-web2-1   Started                                                0.1s
+    ✔ Container nginx-nodejs-redis-nginx-1  Started
+    ```
+
+2. If you look at the Docker Dashboard, you can see the containers and dive deeper into their configuration.
+
+
+    ![A screenshot of Docker Dashboard showing how to delete the multi-container applications](images/delete-multi-container-apps.webp?border=true)
```

Comment:
Verify that this is the correct image.

Reply:
A Good catch. Added the right images at the right location.